test/playback-watcher.test.js

```diff
@@ -1111,6 +1202,8 @@ QUnit.module('PlaybackWatcher isolated functions', {
       tech: {
         on: () => {},
         off: () => {},
+        one: () => {},
+        paused: () => true,
```

Comment:
I think it may make more contextual sense for this to be false for these cases

```suggestion
        paused: () => false,
```

Reply:
I guess you're right. I felt weird about defaulting to a playing state but since this is the playlist-watcher and only executes during playback it seems to make sense